Gemfile

```diff
@@ -127,6 +127,7 @@ group :development do
   gem "i18n-tasks", "~> 1.0.11" # Helpers to find and manage missing and unused translations
   gem "listen", "~> 3.7", require: false # Helps 'listen' to file system modifications events (also used by other gems like guard)
   gem "memory_profiler", "~> 1.0", require: false # Memory profiling routines for Ruby 2.3+
+  gem "ruby-lsp", "~> 0.16.2", require: false
```

Comment:
This should not be necessary. Upon devcontainer launch, vscode lsp extension will install `ruby-lsp`. You can see it working in the `output` tab 

![image](https://github.com/forem/forem/assets/15793250/62836e00-f46f-4ddd-ae88-81eb33c9817e)
![image](https://github.com/forem/forem/assets/15793250/506cb61d-1f18-43aa-a5f7-29d2d8a71f77)